.github/workflows/python-package.yml

```diff
@@ -38,8 +38,8 @@ jobs:
       run: |
         # stop the build if there are Python syntax errors or undefined names
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
I wonder why only specific codes trigger errors. Since there are no flake8 issues currently (after I've adjusted `max-complexity`) we could remove this white listing.

Reply:
Historic reasons I guess. This is in fact the default that GitHub provides for a python package workflow. I am not sure why. But if we can demand higher standard, that is probably a good idea.